### ISSUE TITLE
Stylus not working

### ISSUE THREAD

**Issue by AshotN** [preactjs/preact-cli#938]
**Do you want to request a _feature_ or report a _bug_?**
Not sure if this is a bug, or feature request frankly.

**What is the current behaviour?**
```
Module build failed (from ../node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ../node_modules/preact-cli/lib/lib/webpack/proxy-loader.js):
TypeError: ./src/style/index.styl:20:2
   16| }
   17| 
   18| #app {
   19|  height: 100%;
   20| }
--------^
   21| 

Path must be a string. Received [ './node_modules',
  './node_modules/preact-cli/node_modules' ]

```

**If the current behaviour is a bug, please provide the steps to reproduce.**
`yarn serve`

**What is the expected behaviour?**
For Stylus to build

**If this is a feature request, what is motivation or use case for changing the behaviour?**
The README says there is Stylus support.

**Please mention other relevant information.**
I installed `stylus stylus-loader`

Please paste the results of `preact info` here.
```
preact info
         ▄▄
     ▄▄▓▓▓▓▓▓▄▄
  ▄█▀▀█▓▓▓▓▓▓▓▀▀█▄▄
▐▓▌▐▓▓▓▒▄ ▀▄▄▓▓▓▌▐▓▌ 
▐▓▓▄▀▓▀ ▄▓▓▄▄▀▓▓ ▓▓▌ 
▐▓▓▓▌ ▒▓▌  ▐▓▓  ▓▓▓▌ preact-cli 2.2.1
▐▓▓ ▒▓▄▄▀▓▓▀ ▄▓▓ ▓▓▌
▐▓▌▐▓▓▓▀▀▄▄▀▀▓▓▓▌▐▓▌
  ▀█▄▄▒▓▓▓▓▓▓▒▄▄▒▀
      ▀▓▓▓▓▓▓▀▀
         ▀▀
For help with a specific command, enter:
  preact help [command]

Commands:
  create [template] [dest]  Create a new application.
  build [src] [dest]        Create a production build in build/
  watch [src]               Start a development live-reload server.
  serve [dir]               Start an HTTP2 static fileserver.
  list                      List all official templates

Options:
  -h, --help  Show help                                                [boolean]

Unknown argument: info
```

**Comment by razdvapoka** [comment]
This happens because on [this](https://github.com/preactjs/preact-cli/blob/2cba92cd74a59fba2b93bab1fabfa8a593408b11/packages/cli/lib/lib/webpack/webpack-base-config.js#L219) line the paths are wrapped in an additional array. I was able to fix it  manually through a custom webpack config. I put the following code in the `preact.config.js` file:

```
export default {
  webpack(config, env, helpers, options) {
    const stylusRule = config.module.rules.find(
      rule => String(rule.test) === String(/\.styl$/),
    );
    stylusRule.use[0].options.options.paths =
      stylusRule.use[0].options.options.paths[0];
  },
};
```

**Comment by rschristian** [comment]
Looks to have been fixed in #1152
